runtime/queries/c/highlights.scm

```diff
@@ -1,65 +1,102 @@
-(storage_class_specifier) @keyword.storage
-
-"goto" @keyword
-"register" @keyword
-"break" @keyword
-"case" @keyword
-"continue" @keyword
-"default" @keyword
-"do" @keyword
-"else" @keyword
-"enum" @keyword
-"extern" @keyword
-"for" @keyword
-"if" @keyword
-"inline" @keyword
-"return" @keyword
-"sizeof" @keyword
-"struct" @keyword
-"switch" @keyword
-"typedef" @keyword
-"union" @keyword
-"volatile" @keyword
-"while" @keyword
-"const" @keyword
+[
+  "sizeof"
+] @keyword
+
+[
+  "enum"
+  "struct"
+  "union"
+] @keyword.storage.type
+
+[
+  "const"
+  "extern"
+  "inline"
+  "register"
+  "typedef"
+  "volatile"
+  (storage_class_specifier)
+] @keyword.storage.modifier
+
+[
+  "for"
+  "do"
+  "while"
+  "break"
```

Comment:
`break` is for `switch` too.
The examples in the docs suggest only the loop itself is @keyword.control.repeat.
Maybe `break`/`continue` should be `keyword.control`?

Reply:
I think moving them is fine for now. Ideally I'd like to keep `continue` in `@keyword.control.repeat` since it isn't used as part of `switch`'s, and handle `break` as either `@keyword.control.conditional` or`@keyword.control.repeat` depending on whether it's in a `switch` or not, but that would take a bit more care.

Reply:
Sure. I do think `@keyword.control.repeat` should mean something consistent though: of the languages that have queries for it, it looks like ~25 restrict it to the syntax of the loop itself, while ~2 also highlight loop-related control flow like break/continue.